packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -10,6 +10,338 @@
 @import 'spacing';
 @import './vendor/@carbon/elements/scss/themes/themes';
 
+/// forward declaration of variables (requried with sass 2.0.0)
```

Comment:
The variables declared in the mixin attempt to assign and create global variables ... this isn't allowed in sass 2.0.0, so I've moved the declarations here (but left the same assigns in the mixin) ... I'm not sure if they need to be contained within a mixin, and what that would get us.

Reply:
@cal-smith Given I can see that the original logic for `import-once` stuff, may I ask if Sass 2.0 adoption is imminent in your roadmap? Another reason I wanted to ask this question is that `@carbon/themes` also uses this pattern.

Reply:
If I understand the [description here](https://sass-lang.com/documentation/variables#shadowing) and implications of this change correctly, this particular change _should_ be backwards compatible ... for variables I'm not sure the `import-once` logic applies ...

When I was changing this around I initial had all the variables `null`ed out ... somewhat suprisingly this broke several things (since the variables they were using had no value) ... I can dig more into that and see _why_ exactly it was breaking (I believe it was in the typography file)

Reply:
Good point, probably making compile time slower and possible variable overrides. Given neither of them seems substantial, would you want to just flatten contents in `_theme-tokens.scss`?

Reply:
Sure 👍